<ui version="4.0" >
 <class>Client</class>
 <widget class="QMainWindow" name="Client" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>client</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="infoAccount" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>guest</string>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>state</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="infoStatus" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>offline</string>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" >
     <widget class="QToolBox" name="pageBox" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="news" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>313</width>
         <height>266</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>news</string>
       </attribute>
       <attribute name="icon" >
        <iconset>../images/newspaper.png</iconset>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QListWidget" name="newsList" />
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="newsRead" >
              <property name="text" >
               <string>read</string>
              </property>
              <property name="icon" >
               <iconset>../images/newspaper_go.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="services" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>212</width>
         <height>232</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>services</string>
       </attribute>
       <attribute name="icon" >
        <iconset>../images/bricks.png</iconset>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QTabWidget" name="serviceBox" >
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab" >
             <attribute name="title" >
              <string>web</string>
             </attribute>
             <layout class="QGridLayout" >
              <item row="0" column="0" >
               <widget class="QListWidget" name="serviceWebList" />
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2" >
             <attribute name="title" >
              <string>stream</string>
             </attribute>
             <layout class="QGridLayout" >
              <item row="0" column="0" >
               <widget class="QListWidget" name="serviceStreamList" />
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="serviceManage" >
              <property name="text" >
               <string>manage</string>
              </property>
              <property name="icon" >
               <iconset>../images/brick_edit.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="serviceCredit" >
              <property name="text" >
               <string>credit</string>
              </property>
              <property name="icon" >
               <iconset>../images/coins_add.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="serviceAdd" >
            <property name="text" >
             <string>add_new_service</string>
            </property>
            <property name="icon" >
             <iconset>../images/brick_add.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="credit" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>212</width>
         <height>191</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>credit</string>
       </attribute>
       <attribute name="icon" >
        <iconset>../images/coins.png</iconset>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>currently</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="creditCurrently" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="creditBox" >
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <widget class="QWidget" name="web" >
             <attribute name="title" >
              <string>web</string>
             </attribute>
             <layout class="QGridLayout" >
              <item row="0" column="0" >
               <widget class="QListWidget" name="creditWebList" />
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="stream" >
             <attribute name="title" >
              <string>stream</string>
             </attribute>
             <layout class="QGridLayout" >
              <item row="0" column="0" >
               <widget class="QListWidget" name="creditStreamList" />
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="talk" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>313</width>
         <height>266</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>talk</string>
       </attribute>
       <attribute name="icon" >
        <iconset>../images/user_comment.png</iconset>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QTabWidget" name="talkBox" >
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <widget class="QWidget" name="mycontacts" >
             <attribute name="title" >
              <string>mycontacts</string>
             </attribute>
             <layout class="QGridLayout" >
              <item row="0" column="0" >
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QListWidget" name="talkMyContactsList" />
                </item>
                <item>
                 <layout class="QHBoxLayout" >
                  <item>
                   <widget class="QPushButton" name="talkMyContactsAdd" >
                    <property name="text" >
                     <string>add</string>
                    </property>
                    <property name="icon" >
                     <iconset>../images/user_add.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="talkMyContactsDel" >
                    <property name="text" >
                     <string>delete</string>
                    </property>
                    <property name="icon" >
                     <iconset>../images/user_delete.png</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="allcontacts" >
             <attribute name="title" >
              <string>allcontacts</string>
             </attribute>
             <layout class="QGridLayout" >
              <item row="0" column="0" >
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QListWidget" name="talkAllContactsList" />
                </item>
                <item>
                 <layout class="QHBoxLayout" >
                  <item>
                   <widget class="QPushButton" name="talkAllContactsAdd" >
                    <property name="text" >
                     <string>talk_all_contacts_add</string>
                    </property>
                    <property name="icon" >
                     <iconset>../images/user_add.png</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="talkOpen" >
            <property name="text" >
             <string>talk</string>
            </property>
            <property name="icon" >
             <iconset>../images/comment_edit.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection" >
    <property name="title" >
     <string>file</string>
    </property>
    <addaction name="actionSignIn" />
    <addaction name="actionSignOut" />
    <addaction name="separator" />
    <addaction name="actionSignUp" />
    <addaction name="actionRefresh" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>edit</string>
    </property>
    <addaction name="actionOptions" />
    <addaction name="separator" />
    <addaction name="actionAccounts" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>?</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuConnection" />
   <addaction name="menuEdit" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBarAction" >
   <property name="windowTitle" >
    <string>action</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionSignIn" />
   <addaction name="actionSignOut" />
   <addaction name="separator" />
   <addaction name="actionSignUp" />
   <addaction name="actionRefresh" />
   <addaction name="separator" />
   <addaction name="actionQuit" />
  </widget>
  <widget class="QToolBar" name="toolBarHelp" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbout" />
   <addaction name="actionLogo" />
  </widget>
  <action name="actionSignIn" >
   <property name="icon" >
    <iconset>../images/connect.png</iconset>
   </property>
   <property name="text" >
    <string>sign_in</string>
   </property>
  </action>
  <action name="actionSignOut" >
   <property name="icon" >
    <iconset>../images/disconnect.png</iconset>
   </property>
   <property name="text" >
    <string>sign_out</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset>../images/door.png</iconset>
   </property>
   <property name="text" >
    <string>quit</string>
   </property>
  </action>
  <action name="actionSignUp" >
   <property name="icon" >
    <iconset>../images/database_save.png</iconset>
   </property>
   <property name="text" >
    <string>sign_up</string>
   </property>
  </action>
  <action name="actionRefresh" >
   <property name="icon" >
    <iconset>../images/database_refresh.png</iconset>
   </property>
   <property name="text" >
    <string>refresh</string>
   </property>
  </action>
  <action name="actionOptions" >
   <property name="icon" >
    <iconset>../images/wrench.png</iconset>
   </property>
   <property name="text" >
    <string>options</string>
   </property>
  </action>
  <action name="actionAccounts" >
   <property name="icon" >
    <iconset>../images/group_gear.png</iconset>
   </property>
   <property name="text" >
    <string>accounts</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset>../images/help.png</iconset>
   </property>
   <property name="text" >
    <string>about</string>
   </property>
  </action>
  <action name="actionLogo" >
   <property name="icon" >
    <iconset>../images/logo.png</iconset>
   </property>
   <property name="text" >
    <string>logo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="9" />
 <includes/>
 <resources/>
 <connections>
  <connection>
   <sender>newsList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>newsRead</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>62</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel" >
     <x>60</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceWebList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>serviceManage</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>86</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel" >
     <x>71</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceStreamList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>serviceManage</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>91</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>82</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>talkMyContactsList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>talkOpen</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>73</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel" >
     <x>91</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>talkAllContactsList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>talkOpen</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>137</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel" >
     <x>131</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
